README.md

```diff
@@ -108,7 +110,23 @@ All metrics are exported as gauges.
 
 In addition `cloudwatch_exporter_scrape_error` will be non-zero if an error
 occurred during the scrape, and `cloudwatch_exporter_scrape_duration_seconds`
-contains the duration of that scrape.
+contains the duration of that scrape. `cloudwatch_exporter_build_info` contains 
+labels referencing the current build version and build release date.
+
+### Build Info Metric
+
+`cloudwatch_exporter_build_info` is a default cloudwatch exporter metric that contains the current
+cloudwatch exporter version and release date as label values. The numeric metric value is statically
+set to 1. If the metrics label values are "unknown" the build information scrap failed. 
+
+To display the metrics label values in Grafana:
+```
+1. Query for the cloudwatch_exporter_build_info
+2. Set the Legend to the desired label (ex. {{build_version}} or {{release_date}})
+3. Select Stat visualization
+4. Under stat styles select Name
+5. To remove the graph, select Graph Mode None
+```
```

Comment:
```suggestion

```

This changes every few Grafana versions, let's not duplicate Grafana documentation here.